extensions/amp-consent/0.1/test/test-consent-state-manager.js

```diff
@@ -638,6 +672,7 @@ describes.realWin('ConsentStateManager', {amp: 1}, (env) => {
         expect(value).to.deep.equal(
           constructConsentInfo(CONSENT_ITEM_STATE.REJECTED, 'test1')
         );
+        yield instance.update(CONSENT_ITEM_STATE.ACCEPTED, 'test2');
```

Comment:
can we remove the duplicate update? In theory one update is sufficient.

Reply:
Ooops copy + paster error